src/graph/session/GraphSessionManager.cpp

```diff
@@ -256,10 +256,7 @@ void GraphSessionManager::updateSessionsToMeta() {
   // There may be expired queries, and the
   // expired queries will be killed here.
   auto handleKilledQueries = [this](auto&& resp) {
-    if (!resp.ok()) {
-      LOG(ERROR) << "Update sessions failed: " << resp.status();
-      return Status::Error("Update sessions failed: %s", resp.status().toString().c_str());
-    }
+    DCHECK(resp.ok());
```

Comment:
Why do we convince that resp is always valid?

Reply:
https://github.com/vesoft-inc/nebula/pull/5225/files#diff-a9174d4e2b477c826e5e4e23a7bf43ffe74a5853249f4de3aeebf3d95cf70364R286-R288

Here we checked the response from meta service, and this guarantees `handleKilledQueries()` will receive a valid response.

Reply:
Implicit dependent precondition of  one piece of code depends on another piece is not a good design, alghough it do works. Self-contained code is better choice.

Reply:
> Implicit dependent precondition of one piece of code depends on another piece is not a good design, alghough it do works. Self-contained code is better choice.

Makes sense to me. I'll move the check into the lambda functions.